### PR TITLE
Smed 28 2 - ng messages 

### PR DESCRIPTION
**Pull request by C-E-Rios**
Added ngMessages to various templates
Fix i18n typos, and other issues
Tidy up css for displaying error messages (ng-messages)

### CODE REVIEW

**Comment by C-E-Rios** [comment]
**eventList - master**

eventlist-master

**eventList - ngMessages**

eventlist-ngmessages

**Comment by C-E-Rios** [comment]
**checkbox spacing master**
checkboxspacing-master

**checkbox spacing - ngMessages**
checkboxspacing-ngmessages

**Comment by C-E-Rios** [comment]
**checkbox spacing - error - master**
checkboxspacing-error-master

**checkbox spacing - error - ngMessages**
checkboxspacing-error-ngmessages

**Comment by C-E-Rios** [comment]
**event - master**
master-noerror

**event - ngMessages**
ngmessages-noerror

**Comment by C-E-Rios** [comment]
**event - error - master**
master-error

**event - error - ngMessages**
ngmessages-error

File: src/public-booking/templates/event.html [hunk 1]
```diff
@@ -154,7 +154,7 @@ <h1 class="truncate-sm">{{event.chain.name}}</h1>
               </div>
             </li>
 
-            <li ng-show="selected_tickets && bb.basket.questionPrice()">
+            <li ng-if="selected_tickets && bb.basket.questionPrice()">
```
Comment (lukeferguson) [inline src/public-booking/templates/event.html]: Favour ng-show unless you need to control directive invocation.

File: src/public-booking/templates/event.html [hunk 2]
```diff
@@ -383,21 +425,32 @@ <h2 translate="PUBLIC_BOOKING.EVENT.CUSTOMER_DETAILS_HEADING"></h2>
...
-                        class="error-message"
-                        ng-show="(!$first || !use_my_details) && client_ticket_form.attendee_email.$dirty && client_ticket_form.attendee_email.$invalid && booking_form.submitted"
-                        translate="PUBLIC_BOOKING.EVENT.TICKET_ACCORDION.EMAIL_INVALID">
+                        class="col-sm-5">
+                        <input
+                          type="email"
+                          ng-model="item.email"
+                          name="attendee_email"
+                          class="form-control"
+                          required
```
Comment (lukeferguson) [inline src/public-booking/templates/event.html]: Attendee email shouldn;t be a required field

File: src/public-booking/templates/event_list.html [hunk 3]
```diff
@@ -426,17 +429,17 @@ <h3 class="truncate-sm">{{item.description}}</h3>
                       class="btn btn-primary btn-block"
                       ng-disabled="spaces_left == 0 && !bb.company.settings.has_waitlists">
                       <span
-                        ng-show="spaces_left > 0"
+                        ng-if="spaces_left > 0"
```
Comment (lukeferguson) [inline src/public-booking/templates/event_list.html]: Favour ng-show over ng-if

**Pull request merged**